sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/operations/_question_answering_client_operations.py

```diff
@@ -102,6 +102,43 @@ def query_knowledgebase(
         **kwargs  # type: Any
     ):
         # type: (...) -> "_models.KnowledgeBaseAnswers"
+        """Answers the specified question using your knowledge base.
```

Comment:
Shouldn't the method now be `query_knowledge_base`? I'm using `QueryKnowledgeBase` in .NET.

Reply:
hmmm that's a good point - I think this happened because my swagger readme file was redefining the operations IDs of the APIs - and so the capitalization change in the operation was ignored. I will open an issue for this.